doc/zip/README

```diff
@@ -1,10 +1,44 @@
-HTML:
+FromThePage Export Formats
 
-full -- full text of the transcription or translation, with versions, contributors, and comments.
-text -- just the full text of the transcription or translation.
+This archive contains files exported from FromThePage in one or more of the following formats:
 
-Text:
+HTML FORMATS:
 
-verbatim -- full text of the transcription, with versions, contributors and comments.
-searchable -- links removed and line breaks removed so searches can span line breaks.
-expanded -- expanded transcripts and translations replace the verbatim linked subjects with the canonical name of the subject to make it suitable for text analysis.
+full -- Full text of the transcription or translation, with versions, contributors, and comments.
+text -- Just the full text of the transcription or translation.
+transcript -- HTML transcript files only.
+translation -- HTML translation files only (if work supports translation).
+
+PLAINTEXT FORMATS:
+
+verbatim -- Full text of the transcription, with versions, contributors and comments. Line breaks are represented with single newlines, paragraph breaks with double newlines, and page breaks with triple newlines.
```

Comment:
Plaintext exports never contain versions, contributors, or comments -- they only contain the text.  This is true of verbatim, searchable, and expanded formats.